### PR TITLE
9.0: try to debug pipeline errors

### PR DESCRIPTION
**Pull request by skurfuerst**

### CODE REVIEW

**Comment by skurfuerst** [comment]
Result so far: if I only run the "Basic.feature" testcase in CI, it all runs through successfully.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -293,44 +298,44 @@ jobs:
...
-        run: |
-          FLOW_CONTEXT=Testing/Behat ./flow configuration:show
-          FLOW_CONTEXT=Testing/Behat ./flow doctrine:migrationstatus
-
-          FLOW_CONTEXT=Testing/Behat ./flow behat:setup
-          FLOW_CONTEXT=Testing/Behat ./flow doctrine:create
-          FLOW_CONTEXT=Testing/Behat ./flow doctrine:migrationversion --add --version all
-          # TODO: RE-ENABLE AFTER FIXING
-          #bin/behat --stop-on-failure -f progress -c Packages/Neos/Neos.Neos/Tests/Behavior/behat.yml.dist --tags ~@browser
+#      - name: Setup Flow configuration (PGSQL)
```
Comment (skurfuerst) [inline .github/workflows/build.yml]: TODO: comment this in again

File: composer.json [hunk 2]
```diff
@@ -111,10 +111,8 @@
         ],
         "test:behavioral:stop-on-failure": [
             "mkdir -p ../../Build/Behat/; cp -R Neos.ContentRepository.BehavioralTests/DistributionBehatTemplate/* ../../Build/Behat/; cd ../../Build/Behat/; composer install; cd ../../Packages/Neos",
-            "../../bin/behat -vvv --stop-on-failure -f progress -c Neos.ContentRepository.BehavioralTests/Tests/Behavior/behat.yml.dist",
-            "../../bin/behat -vvv --stop-on-failure -f progress -c Neos.ContentGraph.DoctrineDbalAdapter/Tests/Behavior/behat.yml.dist",
             "../../flow doctrine:migrate --quiet; ../../flow cr:setup",
-            "../../bin/behat -vvv --stop-on-failure -f progress -c Neos.Neos/Tests/Behavior/behat.yml.dist --tags ~@browser"
+            "cd Neos.Neos/Tests/Behavior/; ../../../../../bin/behat -vvv --stop-on-failure -f progress -c behat.yml.dist --tags ~@browser"
```
Comment (skurfuerst) [inline composer.json]: we can revert this

File: Neos.Neos/Tests/Behavior/Features/Bootstrap/RoutingTrait.php [hunk 3]
```diff
@@ -168,7 +168,11 @@ abstract protected function getContentRepository(): ContentRepository;
      */
     public function theDocumenturipathProjectionIsUpToDate(): void
     {
-        $this->getContentRepository()->catchUpProjection(DocumentUriPathProjection::class);
+        if ($this->lastCommandOrEventResult === null) {
+            // we just blocked in "The graph projection is up to date"
+            return;
+        }
+        $this->lastCommandOrEventResult->block();
```
Comment (mhsdesign) [inline Neos.Neos/Tests/Behavior/Features/Bootstrap/RoutingTrait.php:175]: dont we need to set it to null here as well?

(like in `theGraphProjectionIsFullyUpToDate`?)


also in general it seem to be better to use 

> The graph projection is fully up to date

instead no?

Comment (bwaidelich) [reply Neos.Neos/Tests/Behavior/Features/Bootstrap/RoutingTrait.php:175]: @skurfuerst did you see that comment?

Comment (skurfuerst) [reply Neos.Neos/Tests/Behavior/Features/Bootstrap/RoutingTrait.php:175]: Ah thanks. I can push a follow up

**Pull request merged**